2-mandatory/4-restaurant.js

```diff
@@ -21,7 +21,24 @@ const MENU = {
 
 let cashRegister = {
   // write code here
-}
+  orderBurger: function (balance) {
+    if (balance >= MENU.burger) {
+      let newBalance = MENU.burger - balance;
+      return newBalance;
```

Comment:
good job, you do not need a new variable here, you can return the value straight away
you can use ternary operator too if you would like